app/webpacker/styles/components/content-cta.scss

```diff
@@ -0,0 +1,12 @@
+.content__left {
```

Comment:
Does this need to be in the `content__left` block? I think we're trying to move more towards a traditional BEM naming convention where possible